.github/workflows/publish_docs.yaml

```
@@ -0,0 +1,54 @@
+name: Publish docs
+
+on:
+  push:
+    branches:
+      - main
+      - v*.*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+  AWS_S3_BUCKET: "s3.hex.pm"
+
+jobs:
+  release_pre_built:
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - otp: 24
+            otp_version: 24.3
+          - otp: 25
+            otp_version: 25.0
+            build_docs: build_docs
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 50
+      - uses: ./.github/workflows/release_pre_built
+        with:
+          otp_version: ${{ matrix.otp_version }}
+          otp: ${{ matrix.otp }}
+          build_docs: ${{ matrix.build_docs }}
+      - uses: gilacost/branch-tag-action@v1.3
+        id: extract
+      - name: Upload Docs to S3
+        if: ${{ matrix.build_docs }}
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_REGION: ${{ secrets.AWS_REGION }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        run: |
+          version=$(echo ${{ env.BRANCH_TAG }} | sed -e 's/^v//g' | sed -e 's/\//-/g')
```

I believe all we need is:

```suggestion
          version=$(echo ${{ github.ref_name }} | sed -e 's/^v//g')
```

that is, no need for the custom action or changing `/` -> `-` (as they'd never happen), but please let me know otherwise!